static/js/modules/helpers.js

```diff
@@ -118,6 +146,28 @@ function buildUrl(path, query) {
     .toString();
 }
 
+function getTimePeriod(electionYear, cycle, electionFull, office) {
+  var durations = {
+    P: 3,
+    S: 5,
+    H: 1
```

Comment:
Okay, I had figured there was a reason/precedent for it, but wasn't sure.  Thank you for clarifying!